fastly/block_fastly_service_v1_logentries.go

```diff
@@ -127,7 +81,7 @@ func processLogentries(d *schema.ResourceData, conn *gofastly.Client, latestVers
 	return nil
 }
 
-func readLogentries(conn *gofastly.Client, d *schema.ResourceData, s *gofastly.ServiceDetail) error {
+func (h *LogentriesServiceAttributeHandler) Read(d *schema.ResourceData, s *gofastly.ServiceDetail, conn *gofastly.Client) error {
```

Comment:
Can we make sure that we use the same approach that you [used for Datadog](https://github.com/terraform-providers/terraform-provider-fastly/blob/53f1b973ce7b15e5baf5119211b9fe1b55aa25ab/fastly/block_fastly_service_v1_logging_datadog.go) throughout (i.e., [this](https://github.com/terraform-providers/terraform-provider-fastly/pull/267/files#diff-e7c883d632aa25f0ce157e91d526c92fR97) becomes `d.Set(h.GetKey(), ...)`.

Reply:
Similarly, here - https://github.com/terraform-providers/terraform-provider-fastly/pull/267/files#diff-e7c883d632aa25f0ce157e91d526c92fR24